pytorch_lightning/trainer/trainer.py

```diff
@@ -80,6 +81,8 @@
     'please use torch.distributed.ReduceOp instead'
 )
 
+_NEW_LOOP = True
```

Comment:
I'm assuming this is kept as an opt out mechanism.

But how are you making sure that `_NEW_LOOP=False` works? Do you plan to run all CI manually before the PR is merged? And do that for all the next PRs in this chain?

Reply:
NEW_LOOP = True is true right now so it will run tests with the new loops. 
Yes I was testing always with that setting never setting it to False. 

The guard here is simply to make our life easier re-routing and making the diff in trainer readable. 
The PR 2/N will immediately remove this. we will not need to interact with that variable ever.